overlay/constants.py

```diff
@@ -7,10 +7,15 @@
 DB_PORT = int(os.environ.get("DB_PORT"))
 DB_NAME = os.environ.get("DB_NAME")
 MODELS_DIR = os.environ.get("MODELS_DIR")
-
+# username = os.environ.get('ROOT_MONGO_USER')
+# password = os.environ.get('ROOT_MONGO_PASS')
+username = "root"
+password = "rootPass"
```

Comment:
Shall we make these uppercase and rename to `DB_USERNAME` and `DB_PASSWORD` so that they are consistent with other constants?

Reply:
Definitely! I am going to change how this works later. This should be added to a file that is in the gitignore and have these variables populated that way, so this will be changing ASAP.